.github/workflows/build-tag.yml

```diff
@@ -0,0 +1,73 @@
+name: Build and Deploy Catalog from Tag
+
+on:
+  push:
+    tags:
+      - "v*"
```

Comment:
Do we really need to restrict on this?

Reply:
this is what we have on all the other repositories.
the intention is to build a container image only from meaningful tags (i.e. in other repo we have other tags that are not supposed to lead to a release)

Reply:
I left this selector as it is what we have on all the other  repos, we can revisit this later so we can have the same behavior on all the repos